### ISSUE TITLE
Research: MSD Dashboard, model which agencies would most improve RT and accessibility coverage

### ISSUE THREAD

**Issue by edasmalchi** [cal-itp/data-infra#937]
## Question

If we were able to support a select number of agencies in providing GTFS accessibility data and GTFS-RT, which would make the biggest impact in overall coverage statewide (as measured in #585 and #586)?

## Metrics

* estimated coverage increase by agency

## Data sources

* Data Warehouse (GTFS/RT)
* Census (block level from 2020?)

## (Data Servicess Team to Copy and Fill Out Below)

The QuVR MD template below will be filled out by a member of the data services team.
This allows us to describe the request, in a way that is easy to hand-off for analysis.
After the research phase, we will sync with the asker to figure out if the metric and dashboard pieces are needed.

Before starting research:

* **Question**: Question written as a single sentence.
* **View**: E.g. `views.gtfs_schedule_fact_daily_feed_files`.
* **Research**:
  - How should the results be presented?
  - When are they needed by?

After reviewing research with the asker:

* **Metric**: what specific calculations are needed?
* **Dashboard**: where should we put the result?

**Comment by edasmalchi** [comment]
After talking with @e-lo and Taylor, updating bus service buffer distance to 0.5mi

Next steps:

- add detail to map, to extent possible
- map additional coverage after meeting MSD milestones

**Comment by holly-g** [comment]
First iteration complete, new ticket will be open to track for future iteration

**Comment by evansiroky** [comment]
@holly-g where can I view the first iteration?

**Comment by edasmalchi** [comment]
I think I got this issue mixed up with the general coverage metrics and recommended closing, sorry for the confusion!

The work Tiffany and I have done so far is [here](https://github.com/cal-itp/data-analyses/tree/main/msd_dashboard_metric), but I'll go ahead and re-open this one since I don't think it addresses the specific question of modeling potential increases
